css-positioning/absolute.css

```diff
@@ -0,0 +1,44 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  left: 220px;
```

Comment:
this works, but I would consider using the `right` property instead, because you only have to move the emoji a few pixels from the right.

Reply:
I did actually just do right: 10px; at first, but you were talking about positioning from the left side before we started so I swapped. Will trust gut from now on.